meteor/client/ui/RundownView/RundownTiming/RundownTimingProvider.tsx

```diff
@@ -121,16 +159,22 @@ export const RundownTimingProvider = withTracker<
 	{
 		static childContextTypes = {
 			durations: PropTypes.object.isRequired,
+			lowResDurations: PropTypes.object.isRequired,
```

Comment:
What is the point of this split between `durations` and `lowResDurations`?

Reply:
A bit of a naming confusion. `lowResDurations` are for all the components that update in a synced fashion, so that when they re-render for non-timing related reasons, they still hold their time throughout the entire second. I'll fix that.